src/main/kotlin/com/exactpro/th2/httpws/client/Main.kt

```diff
@@ -256,6 +408,7 @@ data class WebSocketSettings(
         TEXT {
             override fun send(client: IClient, data: ByteArray) = client.sendText(data.toString(UTF_8))
         },
+        @Suppress("unused")
```

Comment:
Please, remove it... If we don't use it right now it does not mean we should suppress the warning. I would better have a warning then forget about this enum value when I looking for unused code (if I am doing refactoring or something)